features/division.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Division
+  As a mathematician
+  I want my calculator to divide numbers
+  So I can know their dividend
+
+  Scenario: Dividing numbers to get a dividend
+    When I divide two numbers
+    Then I get the dividend
+
+  Scenario: Dividing zero
+    When I divide a number by zero
+    Then the dividend is infinity
```

Comment:
This is not the requirement. The acceptance criteria is that an error will be raised when diving a number by zero.